tests/kernel/iop/messagebox/mbx-common.c

```diff
@@ -0,0 +1,32 @@
+#include <thbase.h>
+#include <thmsgbx.h>
+#include "mbx-common.h"
+
+u32 getThreadPriority(s32 threadId) {
+	iop_thread_info_t threadInfo = {};
+	ReferThreadStatus(threadId, &threadInfo);
+	return threadInfo.currentPriority;
+}
+
+void printMbx(s32 mbxId) {
+	iop_mbx_status_t mbxInfo;
+	s32 result = ReferMbxStatus(mbxId, &mbxInfo);
```

Comment:
Do the "reserved" values have anything interesting?  I've found that so-called reserved values can often be important, at least in pspsdk.

Also, any reason to log the attr/option values, just in case they vary?

-[Unknown]